### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD006.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#10] [General]
# https://blog.3place.cn/3place/fsd/FSD006.html

https://blog.3place.cn/3place/fsd/FSD006.html

**Comment by picklecai** [comment]
那里已是夜色深沉，
它以不祥、神圣的恐怖，
唤醒我们善良的灵魂。

善良需要恐怖来唤醒，真是没有想到的。再一想，确实：如果没有负面的“打击”，我们如何觉察到正面的情感？白天不能看到灯光的温暖，只有夜里才能看得到。

**Comment by picklecai** [comment]
在这斗室之中，
重新亮起亲切的灯光，
在认识自己的内心之中，
就觉得胸中豁然开朗。
理性又在开始抬头，
希望又在开始出现。

如果一直在这里，没有编号，就没有这种“豁然开朗”。
如果出去了，没有回来，缺少这里的熟悉感，也没有这种“豁然开朗”。
所以，读万卷书后，有一个行万里路。行万里路后，应该还有个：间歇、放松、思考，或者，什么都不干。

**Comment by picklecai** [comment]
太初有言
太初有思
太初有力
太初有为

**Reply by picklecai** [reply]
我不能把言语估计得这样高，
光靠思想就能创造一切？

《思维和语言》，语言真的是太有力量了。我站1。

**Comment by picklecai** [comment]
世人常嘲笑
自己不懂的大道，
善与美常使他们困恼，
他们就对此啧有烦言；

认知就是如此，有什么办法呢？ 夏虫不可语冰，是有道理的。

**Comment by picklecai** [comment]
不管我抱有多大的心愿，
胸中再也涌不出满足之情。

不知足，是这首诗的提线人。

**Comment by picklecai** [comment]
对超自然力要努力尊重，
而且还要渴望“启示”。

既然已知的不能解决问题（不能满足自己），那就把目光投向未知。这是很自然的选择。一切未知，都被视为“超自然力”。

**Comment by picklecai** [comment]
浮士德：请问大名？
梅非斯特：这问题微不足道，
先生对言语总是非常藐视，
总是趋避一切外表，
而只探讨深奥的本质。

特德姜写过，西方对“名”（Name）的看重。问名字，不答名字，是否也反应了梅菲斯特对“名”有一定的畏惧？

**Reply by picklecai** [reply]
先生对言语总是非常藐视，
总是趋避一切外表，
而只探讨深奥的本质。

这是探索、求知、求源，追根溯源者的本能。梅菲斯特打算用这个来糊弄过去被问名。

**Comment by picklecai** [comment]
无生无始者，
无可呼名者，
充塞天地之间者，
横遭枪扎者。

前面三句，既适用于神也适用于魔。
最后一句，如果加上献祭的因素，那神也是可以的。不过以村妇骂人的习惯看，用了最后一句，直觉上遇到的不是什么好人。

那么这一段，是浮士德说出来的台词呢？还是他心里默默的度量？

**Comment by picklecai** [comment]
那种力的一部分
常想作恶，反而常将好事作成。

这个代称，好像女生说那个来了一样。所以，知识的诅咒来了。

**Comment by picklecai** [comment]
生成的一切
总应当要归于毁灭；
所以最好，不如不生。

这似乎是仇视社会者常有的台词，他们在这台词下毁天灭地，其实只是为了弥补某个童年的缺陷。
那么，梅菲斯特的童年缺陷是什么呢？

**Comment by picklecai** [comment]
人类，愚蠢的小宇宙，他们总是
把他们自己当作整体；
我是部分的一部分，部分原本是大全。

如果用分形理论来理解，一下子就通了。

**Comment by picklecai** [comment]
我是黑暗的一部分，光本来生于黑暗。

我一直觉得，黑暗生于光明。有了光，才有了阴影，有了黑暗。《讨厌黑暗的席奶奶》也是这么想的吧？
不过他说得有道理，茫茫宇宙，黑暗森林，光只是特例。我之所以把特例当普遍，是因为我就生在这样的特例中——没有光，就没有生命，即使那些能避光生长的生命，究其根源，也是因为光。

**Comment by picklecai** [comment]
我是常在否定的精灵！

PUA鼻祖？

**Comment by picklecai** [comment]
光发自物体，赋予美丽的外形，却又被物体将去路阻挡。

这句话好神奇。换一个说法：
全宇宙最快的速度，能被一个小小的物体轻易拦住去路。（这个说法，可以写多少奥特曼故事？）
所谓“一叶障目，不见泰山”是也。

**Comment by picklecai** [comment]
我弄清你的伟大的任务！
你不能大规模毁灭万物，
只得先从小规模开始。

这是如何从上文推出来的？不能理解。

**Comment by picklecai** [comment]
对付人和禽兽，这些该死的混蛋，简直没有办法可想。不知有多少已被我埋葬！可是却依然有新鲜的血液在循环。

种族的力量，传承的力量，精神的力量！

**Comment by picklecai** [comment]
恶魔和幽灵有一个规律：走进走出必须打从同一个地方。
走进时自由自在，走出时却是奴隶。

情节的走向，全看规则的制订。就像孩子给一张卡略加技能，就能轻松让我这菜鸟赢他一样。

**Comment by picklecai** [comment]
地狱也有它的法治？
我觉得很好，可以跟阁下订约。

由于梅菲斯特提到了规律，浮士德马上就联想到订约。这么显而易见的推论，只有孩子才能做到。成年人，受了多年束缚（魔鬼是不讲道理的），根本跨不过这一小步。所以，浮士德的特点又增加了一条：保持了孩童的基本思考能力。
